apps/website/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Typography } from '@devlaunchers/components/components/atoms'
+import { TextWrapper, PicWrapper, CheckedWrapper, OnboardingCardLayer, CheckedSVG, IconImg } from './StyledOnboardingCard'
+export default function OnboardingCard({
+    iconImg,
+    title,
+    subtitle,
+    checked,
+}) {
+    console.log({ iconImg });
+    console.log({ title });
+    console.log({ checked });
+    return (
+        <OnboardingCardLayer checked = {checked}>
+            {/* <img src={iconImg} alt="icon" /> */}
+            <PicWrapper>
+            <TextWrapper>
```

Comment:
This "TextWrapper" is not needed here. You can add your paddings to the "PicWrapper" styled component. You can also utilise:

-` gap: 10px;
  gap: 10px 20px; /* row-gap column gap */
  row-gap: 10px;
  column-gap: 20px;` css properties for flex boxes.